lib/index.js

```diff
@@ -155,13 +157,15 @@ Queue.prototype.addItem = function(item) {
  * @param {Mixed} item The item to retry
  * @param {Number} attemptNumber The attempt number (1 for first retry)
  * @param {Error} [error] The error from previous attempt, if there was one
+ * @param {String} [id] The id of the queued message used for tracking duplicate entries
  */
-Queue.prototype.requeue = function(item, attemptNumber, error) {
+Queue.prototype.requeue = function(item, attemptNumber, error, id) {
```

Comment:
I dont believe `requeue` is a public method, and most people shouldnt use it, but feels like we should probably respect the order just in case